engine/pom.xml

```diff
@@ -59,6 +59,7 @@
     <dependency>
       <groupId>io.zeebe</groupId>
       <artifactId>zeebe-msgpack-core</artifactId>
+      <version>0.23.0-SNAPSHOT</version>
```

Comment:
No need to define the version in the `/engine/pom.xml`. All versions are defined in the `/bom/pom.xml`.

```suggestion
```